rpg/commands.py

```diff
@@ -103,3 +103,23 @@ async def travel(ctx, *, area_name):
             await ctx.send("Your level isn't high enough for that area!")
     else:
         await ctx.send("Travel failed.")
+
+
+@parser.command(help_text="create an rpg item listing on the guild market", usage_text="?sell quantity unit_price item")
+async def sell(ctx, price, quantity, *, item_name):
```

Comment:
Just noticed that the order of the `quantity` and `price` arguments are different in the function vs. the usage text – which was intended?